### PR TITLE
tree-wide: do not shadow the global var timezone

### PR DESCRIPTION
**Pull request by phomes**

### CODE REVIEW

**Comment by zonque** [comment]
Might be a matter of taste, but I don't like leading underscores in variable names that much. Maybe `tz` would be a saner option. @poettering?

**Comment by dvdhrm** [comment]
Yeah, leading underscores is for kernel developers. We usually try to find fitting names, as we don't have that weird 80ch line limit. Otherwise, looks good to me.

**Comment by poettering** [comment]
We do prefix some bits with single underscores, and that's OK. Double underscores however are crazy and standard says that that namespaces is owned by the compiler, not projects...

We do use underscores in some places where we return a value call-by-ref and we prepare that value in some variable. There, we usually call the actual parameter "_ret" and the preparation variable "ret", and then eventually just copy ret into *_ret when we are done... Deriving a rule from that: for each variable `_foo` we should have a variable `foo` that it is preparing for. But we should not use underscores to safely shadow system variables really.

Long story short, I'd also prefer if these local variables would just be called `tz` or so...

**Comment by phomes** [comment]
Patch updated in #1108
